containers/cassandra-docker/src/main/docker/cassandra.yaml

```diff
@@ -1,3 +1,20 @@
+#
+# Copyright 2014-2015 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
Do not blindly add copyright notices to files like this.

I do not purposely add the copyright to the top of this file as it comes from the Cassandra binary. We are just modifying the file for our purposes.

See https://github.com/hawkular/hawkular-metrics/blob/master/pom.xml#L302